### ISSUE TITLE
The provided link for the live site does not work

### ISSUE THREAD

**Issue by michael-rivera-florencehc-com** [artilleryio/report-viewer#11]
When going to https://reportviewer.artillery.io/, It does not work. It also has a TLS error

**Comment by michael-rivera-florencehc-com** [comment]
![image](https://user-images.githubusercontent.com/94843926/169113622-355995a1-ff5c-4cda-90fb-65a0b17134e7.png)

**Comment by thxmike** [comment]
![image](https://user-images.githubusercontent.com/2818218/169114195-f86b15a0-a125-411b-a1d7-77b57176b2ec.png)

**Comment by cfryerdev** [comment]
For now you can use http://www.aioreportviewer.com/

**Comment by alramalho** [comment]
Damn I was expecting a bit more from a system claiming to help run "invincible production systems", at least the README to be up to date
(the link is still broken 🤦 )

**Comment by cfryerdev** [comment]
Yeah this is deprecated and I no longer have access to update it, nor do i own the subdomain :( wish things were different
